### PR TITLE
adding hf-gradio-hackathon page

### PR DESCRIPTION
**Pull request by Vaibhavs10**

### CODE REVIEW

**Comment by hypha** [comment]
Could we list this after Trans*Code? I want to put all the workshop days events first.

**Review by hypha — CHANGES_REQUESTED** [review]

File: data/pages-content/hf-gradio-hackathon.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+EuroPython ([see here](https://ep2022.europython.eu))
+
+# WHO CAN JOIN?
+Anyone with a EuroPython Conference or Combined Ticket. Just join the Hugging Face organization: [https://huggingface.co/organizations/EuroPython2022/share/JsnIYMRMwWreVmnLQLpPJfqZBhtKKZnMGV](https://huggingface.co/organizations/EuroPython2022/share/JsnIYMRMwWreVmnLQLpPJfqZBhtKKZnMGV) and show up at the conference.
+ 
+# WHAT DO I NEED TO BRING?
+You will need a computer with internet access.
+ 
+# WHAT DO I NEED TO KNOW?
+Should have existing Python and machine learning knowledge. No Gradio or Hugging Face experience is necessary.
```
[the remark this answers is not in the document]
Comment (Vaibhavs10) [reply data/pages-content/hf-gradio-hackathon.md]: Good catch, my latest commit addresses this. Feel free to resolve this if you're happy.

File: data/pages-content/hf-gradio-hackathon.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+# Build cool demos!
+ Are you interested in machine learning or already have a cool machine learning model you want to create a demo for? Join our [Gradio hackathon](https://huggingface.co/EuroPython2022) and submit and host your demo. For context, Gradio is a Python library that allows developers to demo machine learning models with a friendly web interface so that anyone can use it ([learn more](http://gradio.app)). Browse through some existing demos on Hugging Face spaces. Join the Hugging Face organization [here](https://huggingface.co/organizations/EuroPython2022/share/JsnIYMRMwWreVmnLQLpPJfqZBhtKKZnMGV) to attend the hackathon.
+ 
+ 
+# WHEN?
+Wednesday, 13th July 2022 - Sunday, 17th July 2022 (time and room TBD).
+ 
+# WHERE?
+EuroPython ([see here](https://ep2022.europython.eu))
```
[the remark this answers is not in the document]
Comment (Vaibhavs10) [reply data/pages-content/hf-gradio-hackathon.md]: Fixed

File: data/links.json [hunk 3]
```diff
@@ -65,6 +65,10 @@
           "name": "Django Girls",
           "path": "https://djangogirls.org/en/dublin/"
         },
+        {
+          "name": "Hugging Face Gradio Hackathon",
```
[the remark this answers is not in the document]
Comment (Vaibhavs10) [reply data/links.json:73]: Yes, I somehow thought it was alphabetical.

File: data/pages-content/hf-gradio-hackathon.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
+---
+title: 🤗 Hugging Face Gradio Hackathon 🤗
+subtitle:
+  Come Join us at EuroPython from July 13th to 17th for Gradio’s first ever in person & online hackathon! Build and host machine learning demos using Gradio and Hugging Face to participate.
```
[the remark this answers is not in the document]
Comment (Vaibhavs10) [reply data/pages-content/hf-gradio-hackathon.md:4]: Doneso.

**Review by hypha — APPROVED** [review]
I added one more URL for the demos. All look good.

**Pull request merged**